libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/UrlUtils.java

```diff
@@ -291,6 +291,26 @@ public static String appendUrlParameters(String url, Map<String, String> paramet
         return uriBuilder.build().toString();
     }
 
+    /**
+     * Checks if the url point to the home page of the site
+     *
+     * @param urlString the string url
+     * @return true for home page
+     */
+    public static boolean isHomePage(@Nullable String urlString) {
+        if (urlString != null) {
+            Uri uri = Uri.parse(urlString);
+            String path = uri.getPath();
+            return path != null && (path.equals("") || path.equals("/"));
+        }
+        return false;
+    }
+
+    public static @Nullable String appendToPath(@Nullable String url, @Nullable String path) {
```

Comment:
One question that I have: can `post.getSlug()` be `null`, and if so, do we need to guard here, and perhaps return the passed `url` unchanged?